docs/.vitepress/vite.config.ts

```diff
@@ -93,7 +93,7 @@ export default defineConfig({
             } as any
 
             try {
-              const jiti = JITI(id, {
+              const jiti = (JITI as any)(id, {
```

Comment:
This was erroring after this change, but I'm unsure why.

Reply:
Because of this: https://github.com/unjs/jiti/pull/244